src/test/java/se/sundsvall/webmessagecollector/service/MessageServiceTest.java

```diff
@@ -67,16 +81,85 @@ void getMessages_EmptyList() {
 		final var result = service.getMessages("someFamilyId");
 		// Assert & Verify
 		assertThat(result).isNotNull().isEmpty();
-		verify(repository).findAllByFamilyId(anyString());
-		verifyNoMoreInteractions(repository);
+		verify(messageRepository).findAllByFamilyId(anyString());
+		verifyNoMoreInteractions(messageRepository);
 	}
 
 	@Test
 	void deleteMessages() {
 		//Act
 		service.deleteMessages(List.of(1));
 		//Verify
-		verify(repository).deleteAllById(anyList());
-		verifyNoMoreInteractions(repository);
+		verify(messageRepository).deleteAllById(anyList());
```

Comment:
Verification should not be on anyList. Store a referense of the list and use same()